pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/25
+Revision: 1
+Status: Draft
+Type: Process
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 
+Post-History: 09-July-2024 
+```
+
+# Abstract
+<a name="abstract"></a>
+This PHEP establishes a new tiering structure to PyHC projects, which will automatically affect PyHC packages once it goes into effect. Included herein is information on requirements for each of the new four tiers of PyHC projects (Gold, Silver, Bronze, and Copper), as well as benefits accrued at each tier. 
+
+# Motivation
+<a name="motivation"></a>
+Currently, PyHC is at a crossroads for how to push forward as a community. There are two main schools of thought—originating from bi-annual meeting discussions, telecon chats, and further sidebar converstaion—with regards to what PyHC is and should be:  
+
+- a basic interpretation where PyHC is a collection, and listing, of open-soure Python packages with a relevance to Heliophysics and space physics, or
+- a standards-based interpretation where PyHC strives for compliance with our set standards, package interoperability, and standardization around one or more tools.   
+
+There is utility and validity to both approaches. A new PyHC package tiering system is intended to find a "best of both worlds" with the two ideas. Older, out-of-date, unmaintained, or publication-specific code could still have a place for listing and findability, while also allowing nuance between other packages that are more robust, trustworthy, maintained, and work toward the standards-based interpretation of being a PyHC package. 
+
+Further, this tiering system also allows users to get a clearer picture on what each PyHC package has to offer, and the state of the package's condition and development. Creation of a PyHC package tiering system also allows for justification for a myriad of benefits, for example, consideration for funding from a community travel fund, or extra help with improving a standards grouping grade.
+
+# Rationale
+<a name="rationale"></a>
+Decisions for tiering levels, requirements for each tier, and benefits accrued at each tier are based on conversations with the community (bi-annual meetings, telecons, etc.), and are listed here as a starting point for more discussion, likely to be refined in the future. Initially, ideas were presented to the community in a pyramid format. To make the differences between tiers more visible and understandable, it has been transformed into a spreadsheet format. 
+
+# PyHC Package Tiering Specifications
+<a name="specification"></a>
+There are four tiers proposed in this PHEP: Copper, Bronze, Silver, and Gold. These would replace the current structure of "Core Packages", "Other Packages", and "Un-evaluated Packages". See the table below for requirements associated with each tier:
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
```

Comment:
TBH, I had to look up what those differences were (I just googled how to make a table in markdown and did it this way). What exactly needs to change? @jtniehof

Reply:
I don't think requiring a html table is a good idea. it would make the source hard to read and almost impossible to comment on inline for review.

Reply:
CommonMark doesn't support tables; they are a GitHub extension to the standard.

Discussions on that in the PHEP1 thread: [1](https://github.com/heliophysicsPy/standards/pull/22#discussion_r1370464042), [2](https://github.com/heliophysicsPy/standards/pull/22#discussion_r1370498307) ... this was not extensively discussed, which I took as "accepted right off" but might have been "lost in the noise".

Practically speaking, I checked and reText doesn't support tables. pandoc with `-f commonmark` doesn't. With `-f markdown` or `-f gfm` it does. (Although with `gfm` the table cells don't wrap, and it runs off the page). vscodium previews it fine.

The options as I see (no particular order) are:

1. Remove the table and present in some other form
2. Use an HTML table instead (I haven't checked that this renders properly in, say, pandoc!)
3. Update / replace PHEP1 and switch to GitHub markdown (and update the [Content-Type header](https://www.iana.org/assignments/markdown-variants/markdown-variants.xhtml))
4. Don't update PHEP1 and just proceed with using GitHub markdown anyhow.

I don't know if we want to take this to a discussion outside of this PHEP.

Reply:
@jtniehof This might be a good question to bring up at the next telecon? My gut reaction is to either do steps 3 or 4. Presenting in anything other than a table seems less intuitive/easy to digest, and I'm hearing that using an HTML table is a bad idea.

Reply:
Discussion in #38